src/tracker-component.js

```diff
@@ -75,24 +92,67 @@ const getUserIdCookie = () : ?string => {
 
 const setRandomUserIdCookie = () : void => {
     const ONE_MONTH_IN_MILLISECONDS = 30 * 24 * 60 * 60 * 1000;
-    setCookie('paypal-user-id', generateId(), ONE_MONTH_IN_MILLISECONDS);
+    setCookie('paypal-user-id', generate.generateId(), ONE_MONTH_IN_MILLISECONDS);
+};
+
+
+export const getJetlorePayload = (type : string, payload : Object) : Object => {
```

Comment:
If this needs to be exported, can we make it a child of `Tracker` rather than hanging a new function off of `paypal`?